src/Components/MyEventPage.js

```diff
@@ -78,6 +79,14 @@ class MyEventPage extends Component {
           })
     }
 
+    toggle = tab => e => {
```

Comment:
I believe double arrow function isn't necessary here since you arent returning a function and the parameter 'e' is not being used

Reply:
````
function toggle(id) {
   returns e => { ... };
}
````

??